mews-api.md

```diff
@@ -709,14 +725,31 @@ There are certain rules that need to be followed in order for Mews to process th
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
-| `email` | `string` | _recommended_ | Email. |
+|`email` | `string` | _recommended_ | Email. |
 | `lastName` | `string` | required | Last name. |
 | `firstName` | `string` | optional | First name. |
 | `telephone` | `string` | optional | Telephone. |
 | `nationalityCode` | `string` | optional | [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) - two letter country code. |
 | `languageCode`| `string`| optional | Language [code](https://msdn.microsoft.com/en-us/library/ee825488) of the communication language of the customer. This language will be used as the default for communication with the customer. |
 | `address` | [`Address`](mews-api.md#address) object | optional | Represents address. |
 
+#### Company
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+|`id` | `string` | optional | Identifier of a company. |
+|`iata` | `string` | optional | IATA number of a travel agency. |
+|`name` | `string` | optional | Name of a company or travel agency. |
+| `contact` | [`Contact`](mews-api.md#contact) object | optional | Company or travel agency contact information. |
+
+#### Contact
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `email` | `string` | _recommended_ | Company email address. |
```

Comment:
this is a general Contact DTO. Should not contain company or travel agency words.

Reply:
check the state of the reservation array in the creation example.
There is incorrect value 2